df.building-vectors.xml

```diff
@@ -379,6 +379,10 @@
             <item-attr name='building' value='Instrument'/>
         </enum-item>
     </enum-type>
+
+    <df-other-vectors-type type-name='buildings_other' index-enum='buildings_other_id' item-type='building'>
+        <stl-vector name='BED' pointer-type='building_bedst'/>
```

Comment:
What does it mean when df-other-vectors-type has a child? Is it added after the other vectors?

Reply:
It replaces the vector by the same name. Only stl-vector is allowed as a child of df-other-vectors-type.

Reply:
Ok, so it's overriding what the BED vector would otherwise have been? (Why does just this one need to be overridden?)

Reply:
that was my test. the PR now includes all of the vectors that have defined subclasses.